compiler-rt/CMakeLists.txt

```diff
@@ -290,6 +290,9 @@ option(SANITIZER_USE_STATIC_TEST_CXX
   "Use static libc++ for tests." ${DEFAULT_SANITIZER_USE_STATIC_TEST_CXX})
 pythonize_bool(SANITIZER_USE_STATIC_TEST_CXX)
 
+option(SANITIZER_AARCH64_39BIT_VA
+  "Configure sanitizer allocator for 39-bit VA kernel." OFF)
```

Comment:
Maybe drop the "allocator", just "Configure sanitizer for 39-bit VA kernel."? It's cleaner (and keeps the door open for e.g., MSan, where the shadow mappings also need to be changed.)